.github/workflows/build-commit.yaml

```diff
@@ -0,0 +1,76 @@
+name: Build a Daft commit and store the outputted wheel in AWS S3
+
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      commit:
+        type: string
+        description: The commit hash to build
+        required: true
+      machine_type:
+        type: string
+        description: The machine type to use for the build
+        required: true
+  workflow_call:
+    secrets:
+      ACTIONS_AWS_ROLE_ARN:
+        description: The ARN of the AWS role to assume
+        required: true
+    inputs:
+      commit:
+        type: string
+        description: The commit hash to build
+        required: true
+      machine_type:
+        type: string
+        description: The machine type to use for the build
+        required: true
+    outputs:
+      wheel:
+        description: The wheel file that was built
+        value: ${{ jobs.build-commit.outputs.wheel }}
+
+jobs:
+  build-commit:
```

Comment:
let's set the os to ubuntu 20.04, so we always get an older version of glibc for future compatibility of the wheels

Reply:
Good point. I noticed that builds on newer versions of Ubuntu would result in failures later down the line because of a newer version of glibc, leading to dependency mismatches.

Reply:
Updated.